packages/rocketchat-ui/client/views/app/room.html

```diff
@@ -100,6 +100,24 @@
 							{{/unless}}
 							<div class="wrapper {{#if hasMoreNext}}has-more-next{{/if}} {{hideUsername}} {{hideAvatar}}">
 								<ul aria-live="polite">
+									{{#if roomLeader}}
```

Comment:
This is getting hidden when the user chat window is scrolled all the way down, is this intended?
Also, you can see it when you are the leader of the room and it shows the 'Chat Now' button even though when you click on it, it will show an error (can't chat with yourself)